src/valkeymodule.h

```diff
@@ -2159,9 +2159,12 @@ VALKEYMODULE_API void (*ValkeyModule_ScriptingEngineDebuggerFlushLogs)(void) VAL
 VALKEYMODULE_API void (*ValkeyModule_ScriptingEngineDebuggerProcessCommands)(int *client_disconnected,
                                                                              ValkeyModuleString **err) VALKEYMODULE_ATTR;
 
+VALKEYMODULE_API int (*ValkeyModule_ACLCheckPrefixPermissions)(ValkeyModuleCtx *ctx,
```

Comment:
```suggestion
VALKEYMODULE_API int (*ValkeyModule_ACLCheckKeyPrefixPermissions)(ValkeyModuleCtx *ctx,
```
Minor preference to include key in the name.

Reply:
Done